doc/source/raster.rst

```diff
@@ -28,7 +28,19 @@ To select a spatial subset for data download, one must first switch to the polyg
 
 .. figure:: images/draw_polygon_on.png
 
-When in spatial subset selection mode, you can select a rectangular area by doing a left-click and drag. When selecting an area on high resolution datasets (e.g. BC PRISM), there is the possibility that the user interface will not resolve from screen resolution to dataset resolution. If this is the case, you will be prompted. The can usually be alleviated by zooming the map to a higher zoom level when performing your selection. To adjust the opacity of the climate overlay, there is an opacity slider in the lower left hand corner.
+When in spatial subset selection mode, you can select a rectangular area by doing a left-click and drag. When selecting an area on high resolution datasets (e.g. BC PRISM), there is the possibility that the user interface will not resolve from screen resolution to dataset resolution. If this is the case, you will be prompted. The can usually be alleviated by zooming the map to a higher zoom level when performing your selection.
```

Comment:
Excellent. Thanks for making sure that this is documented in the user docs, since the draw icon isn't amazingly obvious.